bundles/org.eclipse.osgi.compatibility.state/src/org/eclipse/osgi/internal/resolver/StateImpl.java

```diff
@@ -114,7 +114,7 @@ public abstract class StateImpl implements State {
 	 * syntax version 2 will not recognize this attribute.
 	 * @deprecated
 	 */
-	@Deprecated
+	@Deprecated(forRemoval = true, since = "2025-12")
```

Comment:
Similar comment, we will not remove this.  This is implemented at runtime with this constant org.eclipse.osgi.container.builders.OSGiManifestBuilderFactory.ATTR_OLD_REPRIVIDE